### DISCUSSION TITLE
Geospatial database

### DISCUSSION THREAD

**Discussion by francipvb** [sqlalchemy/sqlalchemy#6188] [Usage Questions] [answered]
Hello all,

I'm planning to start a project that needs some basic geographical spatial database support to map place markers. It'll need at least to filter by distance and by rectangle bounds.

I read about GeoAlchemy but it is (at least I know) unmaintained.

Is there any maintained solution for this? How much work will be to add a partial feature set for this?

If this is duplicate, please point me to the right place to look at.

Cheers,

**Comment by zzzeek** [comment, marked as answer]
the more recent project is geoalchemy2.   I originally gave them recipes to establish the patterns they were looking for.    I definitely like it being something that other folks are maintaining because it's too much for us to take on here.   

if you just need some recipes for small use cases you can show me what you need and maybe I can re-demonstrate some of those initial recipes.

**Reply by zzzeek** [reply]
it has commits as recent as three days ago so there's some maintenance going on at least:

https://github.com/geoalchemy/geoalchemy2

**Reply by francipvb** [reply]
Thanks!

It seems that the only outdated thing is the documentation...

Cheers,
